### PR TITLE
OSDOCS-1852: Updating example output for Kubernetes 1.21

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-1852

Preview: https://deploy-preview-31586--osdocs.netlify.app/openshift-enterprise/latest/welcome/index.html (everywhere :) )

### CODE REVIEW

File: modules/nw-sriov-app-netutil.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ Currently there are three API methods implemented:
 +
 [NOTE]
 ====
-Exposing hugepages via Kubernetes Downward API is an alpha feature in Kubernetes 1.20 and is not enabled in {product-title}. The API can be tested by enabling the feature gate, `FEATURE_GATES="DownwardAPIHugePages=true"` on Kubernetes 1.20 or greater.
+Exposing hugepages via Kubernetes Downward API is an alpha feature in Kubernetes 1.21 and is not enabled in {product-title}. The API can be tested by enabling the feature gate, `FEATURE_GATES="DownwardAPIHugePages=true"` on Kubernetes 1.21 or greater.
```
Comment (bergerhoffer) [inline modules/nw-sriov-app-netutil.adoc:21]: @jboxman Can you check whether these updates from 1.20 -> 1.21 are appropriate or not for 4.8?

**Review by adellape — COMMENTED** [review]
:+1:

**Comment by bergerhoffer** [comment]
@zhouying7780 Can you please review this PR? Thanks!

File: modules/nw-sriov-app-netutil.adoc (hunk 1 above)
Comment (jboxman) [reply modules/nw-sriov-app-netutil.adoc:21]: @Billy99, what do you think? Thanks!

Comment (Billy99) [reply modules/nw-sriov-app-netutil.adoc:21]: This is already being handled by #30619.
The FeatureGate moved to beta (still disabled upstream) and is now enabled by SR-IOV Operator. This NOTE is being removed.

Comment (bergerhoffer) [reply modules/nw-sriov-app-netutil.adoc:21]: Thank you both! I'll revert this change.

File: modules/rhel-compute-updating.adoc [hunk 2]
```diff
@@ -76,9 +76,9 @@ You must not enable firewalld later. If you do, you cannot access {product-title
...
 NAME                        STATUS                        ROLES    AGE    VERSION
-mycluster-control-plane-0   Ready                         master   145m   v1.20.0
-mycluster-control-plane-1   Ready                         master   145m   v1.20.0
-mycluster-control-plane-2   Ready                         master   145m   v1.20.0
+mycluster-control-plane-0   Ready                         master   145m   v1.21.0
+mycluster-control-plane-1   Ready                         master   145m   v1.21.0
+mycluster-control-plane-2   Ready                         master   145m   v1.21.0
 mycluster-rhel7-0           NotReady,SchedulingDisabled   worker   98m    v1.14.6+97c81d00e
 mycluster-rhel7-1           Ready                         worker   98m    v1.14.6+97c81d00e
 mycluster-rhel7-2           Ready                         worker   98m    v1.14.6+97c81d00e
```
Comment (zhouying7780) [inline modules/rhel-compute-updating.adoc:84]: All the nodes version also should be updated to `v1.21.0`.

Comment (bergerhoffer) [reply modules/rhel-compute-updating.adoc:84]: Good catch thanks - updated!

**Comment by bergerhoffer** [comment]
@zhouying7780 Updated per your feedback, let me know if this looks good now, thanks!

**Pull request merged**